src/devices/Llc200d3sh/DetectionType.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Iot.Device.Llc200d3sh
+{
+    /// <summary>
+    /// Optomax digital liquid level switch detection types
+    /// </summary>
+    public enum DetectionType
```

Comment:
this name suggests the device has more than one mode of detection and not that this represents detection result. I suggest to either rename reading method to something like `IsMaxWaterLevel` or change this to something like this to also accommodate for other similar sensors (perhaps we could have a way of creating specific sensor which would give you pre-configured class)

```csharp
enum WaterLevel
{
    WaterLevelMax,
    WaterLevelMin,
    WaterNotDetected,
}
```

Reply:
also the other example is that I could technically pour other fluid in there and it will show it as Water which further confirms me we should perhaps not even use word "Liquid" instead of water

Reply:
I don't like using the name water in the sensors code as you point out it can detect liquid as well. I am also not sure about the enum containing min and max as it is a simple switch, therefore when water is detected do you use the min or max enum value.  It could also confuse it with sensors which can detect the amount of liquid present I.e. etape

I agree detectiontype is the wrong name

Reply:
As you point out we could rename the reading method and remove the enum altogether.  Although I think the reading method name would be better as something like `IsLiquidPresent`

Reply:
sounds good to me, I like the proposed name